what u got there? a smoothie?

qawery-just-sad opened the pull request:
- max checkpoints/ stages up to 128(fml now plugin is 700+KB)
- can compare start,stage and cp speeds with pr and sr
- deleted old speed limiter(comented out for now)
- MaxSpeed zone will now use newspeedlimiter
- added sql scripts to update and downgrade database (plz use carefully)

qawery-just-sad committed:
what u got there? a smoothie?
- max checkpoints/ stages up to 128(fml now plugin is 700+KB)
- can compare start,stage and cp speeds with pr and sr
- deleted old speed limiter(comented out for now)
- MaxSpeed zone will now use newspeedlimiter
- added sql scripts to update and downgrade database (plz use carefully)

qawery-just-sad commented:
# WARNING! 
Please backup your data! This is a test version of next release 3.2.X (X is a number of patch and no i don't mean roman numeral 10)

To run this version you need to first update your database with script `SurfTimerV3.2.-1/scripts/mysql-files/upgrading/upgrade-3.2.0.sql`, there is also a downgrade script `SurfTimerV3.2.-1/scripts/mysql-files/downgrade.sql`

Here is precompiled plugin with translations and scripts:
[SurfTimerV3.2.-1.zip](https://github.com/qawery-just-sad/Surftimer-Official/files/6649343/SurfTimerV3.2.-1.zip)

(this version still has built in discord messages, that will be fixed after merge)

Kyli3Boi commented on addons/sourcemod/scripting/surftimer/buttonpress.sp:
```diff
@@ -89,36 +92,111 @@ public void CL_OnStartTimerPress(int client)
+				g_iStartVelsNew[client][zgrp][2] = AllSpeed[2];// Z
+
+				int idiff;
+				int guess = g_SpeedMode[client];
```
Why "guess" and not like "iSpeedMode"?

qawery-just-sad replied:
because we are guessing what speedmode client is using, no real reason tbh

Kyli3Boi replied:
xD but its not really a guess if we know what they have selected "g_SpeedMode[client]"

Kyli3Boi commented on scripts/mysql-files/upgrading/upgrade-3.2.0.sql:
```diff
@@ -0,0 +1,97 @@
+
+REPLACE INTO ck_checkpointsnew (steamid, mapname, cp, time, zonegroup)
+SELECT * FROM (
+    SELECT steamid, mapname, 1 AS cp, cp1 AS time, zonegroup FROM ck_checkpoints WHERE cp1 > 0.0
```
There is something wrong with this giving "Unknown column 'cp1' in 'field list'

Kyli3Boi replied:
Forget this... My bad

qawery-just-sad committed:
fix comment

qawery-just-sad commented:
If you have seen issue while upgrading saying:
 ```#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '--ALTER TABLE `ck_playeroptions2` ADD `velcmphud` INT NOT NULL DEFAULT 1 AF...' at line 1``` It's caused by last line in script which should be commented out (proper way to coment out is to use two dashes followed by space so it looks like this `-- TEXT HERE` or just use c multiple line comment `/* TEXT HERE */`) 
Please use fix from this commit 4bc74cb

qawery-just-sad committed:
fix rng

qawery-just-sad commented:
This fixes checkpoint speed msg when you don't have pb yet 1a1cc5d
[SurfTimer.zip](https://github.com/qawery-just-sad/Surftimer-Official/files/6655634/SurfTimer.zip)

qawery-just-sad committed:
N/A if wr == 0

qawery-just-sad commented:
Exactly what it says in commit msg 20b9416
[SurfTimer.zip](https://github.com/qawery-just-sad/Surftimer-Official/files/6655704/SurfTimer.zip)

Kyli3Boi commented:
LGTM

Kyli3Boi reviewed (approved):
LGTM

qawery-just-sad committed:
added time diff when completing map/bonus in prac
Big thanks to @DotDivi
- changed translations to be in line with others

qawery-just-sad committed:
fix sr/pr not showing correct value #49

qawery-just-sad commented:
Compiled version with commits f979111 and 53dc871
[SurfTimerV3.2.-3.zip](https://github.com/qawery-just-sad/Surftimer-Official/files/6672403/SurfTimerV3.2.-3.zip)

qawery-just-sad committed:
update creation of ck_wrcp table

qawery-just-sad committed:
added:
- small hud option (idk may be broken)
- old speed limiter (will work on that later or never)
- in option menu added chat options

qawery-just-sad commented:
Compiled version with commits 9421500 and cbf820b
[SurfTimerV3.2.-4.zip](https://github.com/qawery-just-sad/Surftimer-Official/files/6784339/SurfTimerV3.2.-4.zip)
Sorry for break in updates

qawery-just-sad committed:
oopsie

qawery-just-sad committed:
some fixes
- stage fix
- maxspeed zone now doesn't care about zonegroup

qawery-just-sad commented:
can someone plz check the latest commit 98b48fa

qawery-just-sad committed:
optimititimisimi
plz tell me that `CHANGE` word exists in normall mysql

qawery-just-sad committed:
Merge branch 'alternate-universe' into ReDoOfHealer
